Makefile

```diff
@@ -39,7 +39,7 @@ test:
 	bundle exec rspec
 	# Test that the docker-compose config is valid. This will error if there are errors
 	# in the YAML files, or incompatible features are used.
-	bin/govuk-docker compose config
+	bin/govuk-docker compose config | grep "ERROR" && exit 1
```

Comment:
I believe `bin/govuk-docker compose config | grep -v "ERROR"` would work the same.

Reply:
Nice, thanks